code/src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,47 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class Limelight {
+  /**
+   * Getting limelight values
+   */
+  private NetworkTable m_limelight;
+
+  public Limelight() {
+    m_limelight = NetworkTableInstance.getDefault().getTable("limelight");
+    m_limelight.getEntry("pipeline").setNumber(0);      
+  }
+
+  public boolean ifTargetSeen(){
+    return m_limelight.getEntry("tv").getDouble(0.0) == 1.0;
+  }
+
+  public void LEDState(int state){
+    m_limelight.getEntry("ledMode").setNumber(state);
+  }
+
+  public int getPipeline(){
+    return (int) m_limelight.getEntry("getpipe").getNumber(0);
```

Comment:
if you're going to be able to get the pipeline, you should also be able to set it.  Note that this requires for the limelight to be configured to "Ignore NetworkTables Index" which can be set through the web interface